.eslintrc.yaml

```diff
@@ -1,7 +1,7 @@
-extends:
-  - xo/esnext
-  - ./node_modules/xo/config/plugins.js
 rules:
+  unicorn/prefer-module: off
+  import/extensions: off
+  unicorn/no-array-reduce: off
```

Comment:
**note:** That's necessary because there is no auto fix for this but after looking into the [reasoning for this rule](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/HEAD/docs/rules/no-array-reduce.md) it seems worth to be discussed.

Reply:
Yes, i agree with the harder to read aspect of array reduce. Will take a look at it.

Reply:
done.